docs/iot/includes/ensure-ssh.md

```diff
@@ -1 +1 @@
-Ensure SSH is enabled on your Raspberry Pi. If needed, [refer to *Enable SSH* in the Raspberry Pi documentation](https://www.raspberrypi.org/documentation/remote-access/ssh/).
+Ensure SSH is enabled on your Raspberry Pi. If needed, [refer to *Enable SSH* in the Raspberry Pi documentation](https://www.raspberrypi.org/documentation/computers/remote-access.html#setting-up-an-ssh-server).
```

Comment:
When I click on this link, it redirects to .com instead of .org. I think you need to update all of the links.
```suggestion
Ensure SSH is enabled on your Raspberry Pi. If needed, [refer to *Enable SSH* in the Raspberry Pi documentation](https://www.raspberrypi.com/documentation/computers/remote-access.html#setting-up-an-ssh-server).
```